src/bun.js/node/node_fs.zig

```diff
@@ -317,6 +319,349 @@ pub const AsyncCpTask = struct {
     }
 };
 
+pub const AsyncReaddirRecursiveTask = struct {
+    promise: JSC.JSPromise.Strong,
+    args: Arguments.Readdir,
+    globalObject: *JSC.JSGlobalObject,
+    task: JSC.WorkPoolTask = .{ .callback = &workPoolCallback },
+    ref: bun.Async.KeepAlive = .{},
+    tracker: JSC.AsyncTaskTracker,
+
+    // It's not 100% clear this one is necessary
```

Comment:
hi @Jarred-Sumner  , wanted to know your opinion on comments like these?
are they cool for all code bases or you add these cause you're the owner. 
Thanks for your great work!